bears/general/IndentationBear.py

```diff
@@ -291,14 +291,14 @@ def get_valid_sequences(file, sequence, annotation_dict):
 
             # ignore if within string
             for string in annotation_dict['strings']:
-                if(sequence_position >= string.start and
-                   sequence_position <= string.end):
+                if(gt_eq(sequence_position, string.start) and
```

Comment:
@abhsag24 Hey will this work? `gt_eq` and `lt_eq` aren't defined until later. Correct me if I am wrong.